test/plugins/trigger.test.js

```diff
@@ -889,6 +915,44 @@ describe('trigger tests', () => {
         assert.equal(event.getBuildOf('target').status, 'RUNNING');
     });
 
+    it('[ a, b, c ] is triggered in restarted event when a fails once and then restarts and succeeds before b,c succeeds', async () => {
+        const pipeline = await pipelineFactoryMock.createFromFile('a_b_c.yaml');
+        const event = await eventFactoryMock.create({
+            pipelineId: pipeline.id,
+            startFrom: 'hub'
+        });
+
+        await event.getBuildOf('hub').complete('SUCCESS');
+        await event.getBuildOf('a').complete('FAILURE');
+
+        const restartEvent = await event.restartFrom('a');
+
+        await restartEvent.getBuildOf('a').complete('SUCCESS');
+        await event.getBuildOf('b').complete('SUCCESS');
+        await event.getBuildOf('c').complete('SUCCESS');
+
+        assert.equal(restartEvent.getBuildOf('target').status, 'RUNNING');
```

Comment:
In `[ a, b ] is triggered~` test above there is the following assert, is it not needed here? (Same for the following xit test)
assert.isNull(event.getBuildOf('target'));

Reply:
Add assert.

Reply:
thank you for your fix.